File: matiasmasso/Mat.NET/DT2/DTOCert.vb
Public Class DTOCert
    Inherits DTOBaseGuid

    <JsonIgnore> Property Stream As Byte()
    <JsonIgnore> Property Image As Image
    Property Pwd As String
    Property Ext As String
    Property Caduca As Date
    Property ImageUri As Uri

    Public Sub New()
        MyBase.New()
    End Sub

    Public Sub New(oGuid As Guid)
        MyBase.New(oGuid)
    End Sub

    Public Function memoryStream() As System.IO.MemoryStream
        Return New System.IO.MemoryStream(_Stream)
    End Function
End Class

File: matiasmasso/Mat.NET/DTO/DTOCert.cs
using Newtonsoft.Json;
using System;

namespace DTO
{
    public class DTOCert : DTOBaseGuid
    {
        [JsonIgnore]
        public byte[] Stream { get; set; }
        [JsonIgnore]
        public Byte[] Image { get; set; }
        public string Pwd { get; set; }
        public string Ext { get; set; }
        public DateTime Caduca { get; set; }
        public Uri ImageUri { get; set; }

        public DTOCert() : base()
        {
        }

        public DTOCert(Guid oGuid) : base(oGuid)
        {
        }

        public System.IO.MemoryStream memoryStream()
        {
            return new System.IO.MemoryStream(Stream);
        }
    }
}
